### ISSUE TITLE
BitmapIcon does not work

### ISSUE THREAD

**Issue by kingofthebongo2008** [microsoft/microsoft-ui-xaml#1755]
Hello,
I have tried to put a custom icon into a navigationViewItem 
I get Access Denied in LoadComponent of the page.
Does BitmapIcon works? How i can put a custom icon in NavigationViewITem

            <muxc:NavigationViewItem Content="Designs" Tag="Designs" x:Name="Designs" SelectsOnInvoked="True">
                <muxc:NavigationViewItem.Icon>
                    <BitmapIcon UriSource="ms-appx:///Assets/globe.png"/>
                </muxc:NavigationViewItem.Icon>
            </muxc:NavigationViewItem>

**Comment by ranjeshj** [comment]
Can you please provide a repro project ?

**Comment by kingofthebongo2008** [comment]
Hello, i was preparing the project and it worked. sorry for the
inconvenience.

**Comment by Elmarcotoro** [comment]
Hello,  
I'm testing two projects:
WinUI 3 with Microsoft.WinUI nuget.
WinUI 3 with Microsoft.ProjectReunion nuget.
Both are running in VS 2019 Preview.

I have followed these docs https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon?view=winrt-19041
The image is in a folder called Assets.

I've tried to place a BitmapIcon in:
`
<AppBarButton >
                <AppBarButton.Icon>
                    <BitmapIcon UriSource="ms-appx:///Assets/Test.png"/>
                </AppBarButton.Icon>
 </AppBarButton>
`
and in:
`
<NavigationViewItem>
                    <NavigationViewItem.Icon>
                        <BitmapIcon UriSource="ms-appx:///Assets/Test.png"/>
                    </NavigationViewItem.Icon>
</NavigationViewItem>
`
I cannot get the Icon to display in either case.  

I have tested this in a UWP project and the icon displays without issue.
